src/frontend/web_application/src/scenes/ContactBook/components/TagList/index.jsx

```diff
@@ -63,28 +63,26 @@ const TagList = ({ tags, onTagClick, nbContactsAll, activeTag }) => {
   const tagList = Array.from(new Set(list));
 
   return (
-    <div>
-      <NavList className="m-tag-list" dir="vertical">
+    <NavList className="m-tag-list" dir="vertical">
+      <TagItem
+        title="All contacts"
+        link=""
+        nbContacts={nbContactsAll}
+        key={uuidV1()}
```

Comment:
it's not in a `map`, key is not useful here and can be removed (plus it will trigger a useless re-render of this tagitem)